# RecursionError

This exception is derived from [`RuntimeError`](/exceptions/RuntimeError.md). It is raised when the interpreter detects that the maximum recursion depth (see `sys.getrecursionlimit()`) is exceeded.